pkg/proxy/nftables/helpers_test.go

```diff
@@ -264,6 +284,7 @@ func (tracer *nftablesTracer) matchDestPort(elements []*knftables.Element, proto
 // match verdictRegexp.
 
 var destAddrRegexp = regexp.MustCompile(`^ip6* daddr (!= )?(\S+)`)
+var destAddrsRegexp = regexp.MustCompile(`^ip6* daddr (!= )?\{(.*)\}`)
```

Comment:
```suggestion
var destAddrsRegexp = regexp.MustCompile(`^ip6* daddr (!= )?\{([^}]*)\}`)
```
so if there's another set/map later in the rule you don't accidentally match all the way to the end of that.

Also, the other regexps that do set lookups all have "Lookup" in their name. Maybe `destAddrLookupRegexp`?

(Same comments for `sourceAddrsRegexp`)

Reply:
replaced `.` with `[^}]`, renamed the variables.